packages/esm-service-queues-app/src/patient-queue-metrics/queue-metrics.resource.ts

```diff
@@ -6,7 +6,7 @@ import { startOfDay } from '../constants';
 
 export function useServiceMetricsCount(service: string, location: string) {
   const status = 'Waiting';
-  const apiUrl = `${restBaseUrl}/queue-entry-metrics?service=${service}&status=${status}&location=${location}`;
+  const apiUrl = `${restBaseUrl}/queue-entry-metrics?service=${service}&status=${status}&location=${location}&isEnded=false`;
```

Comment:
Looks good.  We should also address the hard-coded "Waiting" status, but that can be another ticket.